scripts/user/update-judge.ts

```diff
@@ -0,0 +1,301 @@
+  dynamoUser.name = updates.name || dynamoUser.name;
+  dynamoUser.contact = updates.phone
+    ? { phone: updates.phone }
+    : dynamoUser.contact;
+  dynamoUser.isSeniorJudge =
+    updates.isSeniorJudge != ''
+      ? updates.isSeniorJudge.toLowerCase() === 'true'
+      : dynamoUser.isSeniorJudge;
+  dynamoUser.judgeFullName = updates.judgeFullName || dynamoUser.judgeFullName;
+  dynamoUser.section = chambersSection;
+
+  const rawUser = new User(dynamoUser).validate().toRawObject();
+
+  console.log('Updating the Dynamo record ...');
+  await applicationContext.getPersistenceGateway().updateUser({
+    applicationContext,
+    user: rawUser,
+  });
+};
+
+const createUserChambersSectionRecord = async ({
+  applicationContext,
+  chambersSection,
+  userId,
+}) => {
+  await client.put({
+    Item: {
+      pk: `section|${chambersSection}`,
+      sk: `user|${userId}`,
+    },
+    applicationContext,
+  });
+};
+
+const updateDynamoChambersRecords = async ({
+  applicationContext,
+  oldChambersSection,
+  updatedChambersSection,
+  userId,
+}) => {
+  console.log('Chambers section needs to be updated.');
+  console.log(`Adding a record for ${updatedChambersSection}`);
+  // If there is no old chambers section, we only need to add the section record
+  if (!oldChambersSection) {
+    await createUserChambersSectionRecord({
+      applicationContext,
+      chambersSection: updatedChambersSection,
+      userId,
+    });
+    return;
+  }
+
+  // Otherwise, we need to update existing records for every chambers member, including the judge
+  console.log(
+    `Updating members of ${oldChambersSection} to be members of ${updatedChambersSection} ...`,
+  );
+  const chambersUsers: User[] = await applicationContext
+    .getPersistenceGateway()
+    .getUsersInSection({ applicationContext, section: oldChambersSection });
+
+  for (let chambersUser of chambersUsers) {
+    console.log(`Updating ${chambersUser.role} user ${chambersUser.userId}`);
+    chambersUser.section = updatedChambersSection;
+    const rawChambersUser = new User(chambersUser).validate().toRawObject();
+
+    // Update the user record, create the section record, and remove the old section record
+    await applicationContext.getPersistenceGateway().updateUser({
+      applicationContext,
+      user: rawChambersUser,
+    });
+    await createUserChambersSectionRecord({
+      applicationContext,
+      chambersSection: updatedChambersSection,
+      userId: chambersUser.userId,
+    });
+    await client.remove({
+      applicationContext,
+      key: {
+        pk: `section|${oldChambersSection}`,
+        sk: `user|${chambersUser.userId}`,
+      },
+    });
+  }
+};
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises, complexity
+(async () => {
+  const applicationContext = createApplicationContext();
+
+  const currentEmail = process.argv[2];
+
+  const updates = {
+    email: getArgValue('email'),
```

Comment:
A common update that I have seen is being able to update the judges title. So going from Judge -> Special Trial Judge

Reply:
Good call. This should be in add-judge.ts too since we might want to onboard a Special Trial Judge. Working on that now.